servers/rendering/renderer_rd/shaders/effects/copy_to_fb.glsl

```diff
@@ -203,6 +203,14 @@ void main() {
 		color.rgb = normalize(color.rgb * 2.0 - 1.0) * 0.5 + 0.5;
 	}
 
+#ifdef MODE_OCCLUSION_CULLING_BUFFER
```

Comment:
This should just be a flag otherwise we are increasing load times for no reason

Reply:
I've pushed a second commit to attempt this, but it doesn't seem to work on my end.